### ISSUE TITLE
Whoami returns root when invoked from script started with sudo

### ISSUE THREAD

**Issue by Yethal** [mstaal/i3buntu#5]
When script s started with sudo, whoami returns root instead of the username of calling user. Replace with logname for the last two lines to work correctly.

**Comment by mstaal** [comment]
I have tried to update the README to solve the issue. Can you test it?

**Comment by Yethal** [comment]
Yes, this does work. But you can simply replace $(whoami) with $(logname) to achieve the desired effect. You can test that by running following commands
sudo sh -c 'echo $(whoami)' - that will return root
sudo sh -c 'echo $(logname)' - that will return the correct username
